site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -203,6 +203,12 @@ If only one extension has a rule that matches a request, that rule is applied. B
 
 1. If more than one rule matches, the most recently installed extension gets priority.
 
+### Safe rules {: #safe-rules }
```

Comment:
What do you think about removing this section? It feels a bit random as it's not clear why we're mentioning it here. How about merging this into the actual limits section and to not use the term "safe rules" at all?

Reply:
Done! I did leave the mention of safe rules - since we have the constant called `MAX_NUMBER_OF_UNSAFE_DYNAMIC_RULES`, I think we're locked in to documenting the term safe rule.